app/talk/inbox-form.cjsx

```diff
@@ -0,0 +1,64 @@
+React = {findDOMNode} = require 'react'
+talkClient = require '../api/talk'
+UserSearch = require '../components/user-search'
+{Navigation} = require 'react-router'
+{getErrors} = require './lib/validations'
+subjectValidations = require './lib/message-subject-validations'
+messageValidations = require './lib/message-validations'
+
+module?.exports = React.createClass
+  displayName: 'InboxForm'
+  mixins: [Navigation]
+
+  propTypes:
+    user: React.PropTypes.object
+
+  getInitialState: ->
+    validationErrors: []
+
+  validations: (message, subject, recipients) ->
+    userErrors = if (not recipients.length) then ['Messages must have a recipient'] else []
+    subjectErrors = getErrors(subject, subjectValidations)
+    messageValidationErrors = getErrors(message, messageValidations)
+
+    validationErrors = userErrors.concat subjectErrors.concat messageValidationErrors
+    @setState {validationErrors}
+    !!validationErrors.length
+
+  onSubmitMessage: (e) ->
+    e.preventDefault()
+    form = findDOMNode(@).querySelector('form')
+
+    recipient_ids = form.querySelector('[name="userids"]')
+      .value.split(',').map (id) -> parseInt(id)
+      .filter(Number)
+
```

Comment:
Ah yeah (but it'd have to be an array). There could be multiple recipients, but I wanted to limit to one until I get to #738